### PR TITLE
i18n: add Flask-Babel requirement

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

add Flask-Babel requirement

## Testing

vagrant up development # will install from securedrop-requirements.txt

## Deployment

1. Upgrading existing production instances.
2. New installs.

It will install new python modules that will not be used and will not interfere with existing or new deployment.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### CODE REVIEW

**Comment by dachary** [comment]
@heartsucker this is the tiniest pull request I can think of. How does that look ?

**Comment by heartsucker** [comment]
Hey @dachary. Dang, you got that done the same afternoon we had coffee?! I'll take a look at this when I get a chance.

**Comment by dachary** [comment]
> I am slightly less amazed.

As you should :-) But it takes a minute to review. The more substantial series is at https://github.com/dachary/securedrop/commits/wip-i18n-base if you feel like it.

**Comment by dachary** [comment]
a friendly reminder that this is the first of [a few dozens of commits implementing i18n](https://github.com/dachary/securedrop/commits/wip-i18n-base). It would be nice to merge it so we can move to the next commit :-)

**Comment by redshiftzero** [comment]
Hey @dachary - a343b6a looks good. When you get a chance, can you add the next discrete chunk of your internationalization branch - ideally a part that uses this new requirement - to this PR for review?

**Comment by dachary** [comment]
@redshiftzero the first commit using flask-babel is https://github.com/dachary/securedrop/commit/d7a96a51683c77f790b79b86ee07d617741391d9 but it does not only depends on this commit. It adds to manage.py and depends on changes from https://github.com/freedomofpress/securedrop/pull/1850 for

* testing
* running shell commands

File: securedrop/requirements/securedrop-requirements.txt [hunk 1]
```diff
@@ -17,6 +19,7 @@ markupsafe==1.0           # via jinja2
 psutil==5.2.2
 pycrypto==2.6.1
 pyotp==2.2.5
+pytz==2017.2              # via babel
```
Comment (conorsch) [inline securedrop/requirements/securedrop-requirements.txt]: :squint: I'm looking at you, `pytz`, to explain #2070.

**Pull request merged**